src/ruby/yast/y2_exception.rb

```diff
@@ -0,0 +1,15 @@
+require "yast"
+
+module Yast
+  # For passing exceptions through liby2
+  class Y2ExceptionClass < Module
+    def main
+      @exception = {}
+    end
+
+    publish variable: :exception, type: "map"
+  end
```

Comment:
In general change looks OK for me, I just miss more documentation in this module. Especially use cases ( exceptions when calling ruby code from non-ruby parts (includes e.g. also pkg bindings) ), how that variable exception looks like and who can set it. And maybe a paragraph in readme would make sense to point to this.